components/user_settings/advanced/user_settings_advanced.jsx

```diff
@@ -384,7 +391,7 @@ export default class AdvancedSettingsDisplay extends React.PureComponent {
                     <div>
                         <br/>
                         <FormattedMessage
-                            id='user.settings.advance.sendDesc'
+                            id={fmIdForUserSettingsAdvSendDesc}
                             defaultMessage='When enabled, CTRL + ENTER will send the message and ENTER inserts a new line.'
```

Comment:
We'll also want to change the `defaultMessage` on Mac as well since it gets displayed when the current language is missing the translation string. That won't matter for English obviously, but it will be needed for languages that aren't updated as frequently

Reply:
ok. so should I do what i was doing before?   using Utils.isMac()    and changing CTRL for CMD?

Reply:
Nope, you just need 2 variables for each string like
```javascript
let id = t('something');
let defaultMessage = 'Something';
if (Utils.isMac()) {
    id = t('something.mac');
    defaultMessage = 'Something on Mac';
}

return <FormattedMessage id={id} defaultMessage={defaultMessage}/>;
```

Reply:
done!